doc/manual/conf.py

```diff
@@ -56,3 +56,18 @@
 latex_documents = [
    ('index', 'SoundScapeRenderer.tex', project, u'SSR Team', 'manual'),
 ]
+
+def gh_link_role(rolename, rawtext, text, lineno, inliner,
+                     options={}, content=()):
+    from docutils import nodes, utils
+    github_url = 'https://github.com/SoundScapeRenderer/listening-test'
```

Comment:
You are using this role for two different repos.

I guess we'll need two roles now.

Reply:
Indeed. I fixed that.